src/sentry/api/endpoints/group_notes.py

```diff
@@ -47,11 +47,18 @@ def post(self, request, group):
             return Response('{"detail": "You have already posted that comment."}',
                             status=status.HTTP_400_BAD_REQUEST)
 
-        GroupSubscription.objects.subscribe(
-            group=group,
+        subscribe_on_action = UserOption.objects.get_value(
             user=request.user,
-            reason=GroupSubscriptionReason.comment,
-        )
+            project=None,
+            key='subscribe_on_action',
+            default='1',
+        ) == '1'
+        if subscribe_on_action:
```

Comment:
Yeah — I erred here on the side of being consistent, with any action you perform (short of subscribing) preventing you from being subscribed to an issue.

In practicality, I don't think you would want to drive-by comment on an issue without being notified about follow-up comments later. (For what it's worth, I also don't think that most people would want to change issue state, etc. without being notified about changes later, but…)